Do not export internal dependencies in Libs:
(because it's useless, but also because "boost_python" may not be the exact required lib name)

--- CMakeLists.txt~	2016-01-18 10:55:07.000000000 +0100
+++ CMakeLists.txt	2016-06-02 11:52:46.349575911 +0200
@@ -129,8 +129,6 @@
 ENDIF(EIGEN_NUMPY_ALIGNED)
 PKG_CONFIG_APPEND_CFLAGS("-I${PYTHON_INCLUDE_DIRS}")
 PKG_CONFIG_APPEND_CFLAGS("-I${NUMPY_INCLUDE_DIRS}")
-PKG_CONFIG_APPEND_LIBS("boost_python")
-PKG_CONFIG_APPEND_LIBS_RAW(${PYTHON_LIBRARIES})
 PKG_CONFIG_APPEND_LIBS(${PROJECT_NAME})
 
 SETUP_PROJECT_FINALIZE()
